src/views/studio/SignInView/SignInView.tsx

```diff
@@ -22,7 +22,7 @@ import {
 export const SignInView = () => {
   const navigate = useNavigate()
   const { activeChannelId, setActiveUser, memberships } = useUser()
-  const { nodeConnectionStatus } = useConnectionStatus()
+  const nodeConnectionStatus = useConnectionStatusStore((state) => state.nodeConnectionStatus)
```

Comment:
Previously if there was no internet connection all the tiles and sign-in button were disabled. Right now if you disconnect from wifi, you'll still be able to click those.